packages/account/src/Components/form-field-info/form-field-info.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Popover } from '@deriv/components';
+import { TPopoverProps } from '@deriv/components/src/components/types';
+import { isMobile } from '@deriv/shared';
+
+/**
+ * A component that renders a popover with an info icon.
+ *
+ * @param {TPopoverProps} props - Props for the popover component.
+ * @returns React.ReactElement - A React component.
+ */
+export const FormFieldInfo = (props: Omit<TPopoverProps, 'alignment'>) => {
+    const [is_open, setIsOpen] = React.useState(false);
+    const ref = React.useRef<HTMLDivElement>(null);
+
+    React.useEffect(() => {
+        // Close popover when clicking outside of it
+        const handleClickOutside = (event: MouseEvent) => {
+            const target = event.target as HTMLElement;
```

Comment:
so basically here we even can skip type for event, as you tell TS, that event.target is HTMLelement,
is there any chance to avoid using 'as'  in our ts files?

Reply:
This casting is required as click gives us mouse event and target need to be html to distinguish else includes in the very next line gives type error for incompatibility